### PR TITLE
Implementation mongodb

### PR DESCRIPTION
**Pull request by DilanOjeda**
Please check!

### CODE REVIEW

File: CONVERTER-SERVICE/src/middlewares/fileValidator.middleware.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ class FileValidator {
     
           newFile.save()
             .then(() => {
+              req.file.path = uploadSave;
               return next();
             })
             .catch(() => {
```
Comment (stospina) [inline CONVERTER-SERVICE/src/middlewares/fileValidator.middleware.js:49]: Consider throwing customized exceptions maybe for futures tests

**Review by stospina — COMMENTED** [review]
Looks good...

**Review by florpadilla — APPROVED** [review]
It looks good!

**Review by luisdavidparra — APPROVED** [review]
All ok

**Pull request merged**